### ISSUE TITLE
How do I build it into static HTML, CSS, JavaScript assets?

### ISSUE THREAD

**Issue by praveenscience** [ModularCode/modularity-free-admin-dashboard-theme-html#101]
Pretty much that's my question. I am using a PHP based application and I would like to use the current Yarn app in PHP. How should I be converting this into pure HTML, CSS, JavaScript?

**Comment by m10241** [comment]
1. Download a copy of files in your computer and extract the contents of the zip file. 2. Navigate to the extracted folder
Make sure that you have [node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/) installed on your computer
Run the following command:
`npm install`
After it is done, go to the "dist" folder and you will find pure HTML, CSS, and JavaScript files.

**Comment by yanadhorn** [comment]
does this mean i can use ' pure HTML, CSS, and JavaScript files" in my django developement?

**Comment by m10241** [comment]
Yes, you can use those files in combination with any technologies that supports HTML, CSS, and JavaScript! 
I don't personally have any experience developing python web applications, but I assume it is OK!

**Comment by yanadhorn** [comment]
thank you very much i do some research and found what every tech it use to dev. i just build it to html,css,javascript and then i can inject python or django code into it. what a cool.
